packages/core/src/common/externals/NodeJsExternals.ts

```diff
@@ -118,7 +118,9 @@ export const NodeJsExternals: Externals = {
 			return fsp.unlink(toFsPathLike(location))
 		},
 		watch(locations) {
-			return new ChokidarWatcherWrapper(chokidar.watch(locations.map(toPath)))
+			return new ChokidarWatcherWrapper(
+				chokidar.watch(locations.map(toPath), { usePolling: process.platform === 'win32' }),
```

Comment:
`process` is not available in the browser, so a new external method needs to be added to test for the platform

Reply:
we use it above in `showFile` -- is that method just never called in the browser?

Reply:
we already have a separate `BrowserExternals` file which i believe handles this stuff. i think using `process.platform` is fine because it's exclusive to this `NodeJsExternals` file

https://github.com/SpyglassMC/Spyglass/blob/cfa732191f3896cdf8b65cca1a3351ad76e16680/packages/core/src/common/externals/BrowserExternals.ts#L151-L153

Reply:
i'm not sure we've verified if the file-lock issue is even a thing in browsers

Reply:
Oh you are totally right 🤦